triagebot.toml

```diff
@@ -102,6 +102,15 @@ message_on_add = """\
 """
 message_on_remove = "Issue #{number}'s prioritization request has been removed."
 
+[notify-zulip."I-nominated"]
+required_labels = ["T-rustdoc"]
+zulip_stream = 266220 # #rustdoc
+topic = "#{number} nominated"
```

Comment:
Another option: `nominated: #{number}`. I skipped putting the title in the topic because it gets confusing when the issue is renamed (I've experienced this before with WG-prioritization).

Reply:
I like `nominated: #{number}`

Reply:
```suggestion
topic = "nominated: #{number}"
```